### PR TITLE
Merging most up to date code to master

### PR DESCRIPTION
**Pull request by LindseyWeiskopf**
This PR should be merging the most up to date code into the master branch. (Includes all of the commits from post, datastore, and additional-features branches). Hopefully, this will resolve the git issues!

### CODE REVIEW

File: portfolio/src/main/webapp/contact.html [hunk 1]
```diff
@@ -26,10 +27,25 @@ <h1>Contact me here</h1>
...
+      <br></br>  
+      <label for="quantity">Comments shown:</label>
+      <input type="number" id="quantity" name="quantity" min="1" max="20">
 
-      </form>
+      <br></br>
+      <input type="submit" onclick="getComments()"/>
+
+      
+
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/contact.html:41]: nit: Remove the extra blank spaces.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,17 +13,45 @@
 // limitations under the License.
 
 
+function getComments() {
+
+  document.getElementById('greeting-container').innerHTML = "";
+  var commentNum = parseInt(document.getElementById('quantity').value);
+  
+
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/script.js:21]: nit: You can remove one of these blank lines.

File: portfolio/src/main/webapp/contact.html (hunk 1 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/contact.html:41]: Done

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/script.js:21]: and Done!

**Pull request merged**